### PR TITLE
#14: Random IP.

### PR DESCRIPTION
**Pull request by Oscaner**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -169,6 +178,10 @@ async function translate(text, from, to, correct, raw, userAgent, proxyAgents) {
     cookie: globalConfig.cookie
   }
 
+  if (ipProxy) {
+    requestHeaders['CLIENT-IP'] = requestHeaders['X-FORWARDED-FOR'] = randomip.getChineseIp()
```
Comment (plainheart) [inline src/index.js:182]: Thanks for your contribution. It's unbelievable that the bing translate server will simply take the proxy header from the front clients as the real IP! 🤔
Could you please do more tests to confirm this action is practical for bypassing the IP limitation?

Besides, I think it's a bit inappropriate to only use China's IP as this service is not only being used by the Chinese.

Comment (Oscaner) [reply src/index.js:182]: Thanks for your reply. Next step, I will create tests for it, and fix Global's IP instead of China's IP.

Comment (plainheart) [reply src/index.js]: It seems a bit hard for me to reproduce the _426 Too Many Requests_ error. I even increased the requests to 2k. 😕

Comment (losefor) [reply src/index.js]: @plainheart i have the 429 response and can't use it and i want to make a script on 4000 db records and translate

**Pull request closed without merging**